kork/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/ErrorHandlingExecutorCallAdapterFactory.java

```diff
@@ -79,7 +79,7 @@ public static ErrorHandlingExecutorCallAdapterFactory getInstance() {
      * it cannot be handled by this factory
      */
     if (getRawType(returnType) != Call.class) {
-      return null;
+      return new LegacySignatureCallAdapter(callbackExecutor, returnType, retrofit);
```

Comment:
This is pretty compelling.  Thoughts @kirangodishala ?  The javadoc could use an update since this no longer returns null.